src/main/java/cloud/orbit/spring/OrbitSpringConfiguration.java

```diff
@@ -29,50 +29,112 @@
 package cloud.orbit.spring;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.beans.factory.config.AutowireCapableBeanFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 import cloud.orbit.actors.Stage;
-import cloud.orbit.actors.extensions.ActorExtension;
-import cloud.orbit.actors.runtime.Messaging;
+import cloud.orbit.actors.cloner.ExecutionObjectCloner;
+import cloud.orbit.actors.cluster.ClusterPeer;
+import cloud.orbit.actors.extensions.*;
+import cloud.orbit.actors.runtime.*;
 
+import java.time.Clock;
 import java.util.List;
+import java.util.Timer;
+import java.util.concurrent.ExecutorService;
 import java.util.concurrent.TimeUnit;
 
+@SuppressWarnings({ "SpringAutowiredFieldsWarningInspection", "SpringJavaAutowiringInspection" })
```

Comment:
These warnings are coming from my IDE (IntelliJ) with a message:
> Spring Team recommends: "Always use constructor based dependency injection in your beans. Always use assertions for mandatory dependencies."

I think it's quoting [this blog post](https://spring.io/blog/2015/11/29/how-not-to-hate-spring-in-2016).

The reason autowiring fields seemed to be necessary is that I could not find out how to indicate that a constructor-injected dependency is optional.